apps/dashboard/src/utils/semaphoreAPI.ts

```diff
@@ -0,0 +1,61 @@
+import { Subgraph } from "@semaphore-protocol/subgraph"
```

Comment:
Nice 👍 
This model is definitely better than having hooks for everything where all it does is export stateless functions. We can put these two APis under a `data` folder though as they are not `utils`? 🤔 
Anyway, we would need another refactoring here when we introduce SIWE and have same account create/display both on-chain and off-chain groups. The components wont have to worry about which flow it is (on-chain v off-chain) and can call same function to create/get both group types.

Reply:
I renamed that folder to `api`, as they both are basically API. How does it sound?

Ah right, we'll update them 👍🏽

Reply:
Make sense 👍